miditoolkit/midi/__init__.py

```diff
@@ -1,5 +0,0 @@
-from .containers import *
```

Comment:
We could import here all container classes to directly access if from `miditoolkit.`